greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExportRunType.java

```diff
@@ -0,0 +1,8 @@
+package com.greenback.kit.model;
+
+public enum AutoExportRunType {
```

Comment:
AutoExport's right now are just for exports.  Other lambda types we'd likely expose thru other APIs.  I think we can get rid of having any type on this.